dashboard/src/components/attribute/clip-the-middle/che-clip-the-middle.styl

```diff
@@ -8,6 +8,7 @@
     text-overflow ellipsis !important
     overflow hidden
     white-space nowrap
+    padding-right 1px
```

Comment:
checked with **padding-right** - it works not in all cases
here removing **"overflow: hidden**" helps.

Reply:
I'm affraid that overflow:hidden from here is influencing this element too:
file:///home/katka/Obr%C3%A1zky/Screenshot%20from%202018-05-25%2012-57-27.png